pyccel/codegen/printing/fcode.py

```diff
@@ -2884,8 +2916,17 @@ def indent_code(self, code):
                      for line in code]
         decrease = [int(dec_regex.match(line) is not None)
                      for line in code]
-        continuation = [int(any(map(line.endswith, ['&', '&\n'])))
-                         for line in code]
+
+#        def continuation_cnd(line):
+#            def cnd(s):
+#                cond = line.endswith(s)
+#                sline = line.lstrip()
+#                cond = line.endswith(s) and (not sline.startswith('!') or sline.startswith('!$'))
+#                return cond
+#            return cnd
+
+#        continuation = [int(any(map(continuation_cnd(line), ['&', '&\n'])))
+#                         for line in code]
```

Comment:
Can this code be removed or do you think it will be useful later?

Reply:
It can be useful if we add the continuation lines in some print methods, or we allow the user to create the continuation lines by himself

Reply:
Do you think that is likely? I am not sure that it is, or if it is added then I don't think it will be soon.

Given that fact I am not sure that this code is sufficiently complicated to make it worth keeping in comments. What do you think?

Reply:
We have the multiline pragmas, we now ignore the user's continuation lines and merge them all in one line to process them, we might want to keep the user's continuation lines when we print them

Reply:
and if we do we will need to uncomment those lines

Reply:
I understood that. My question is whether these lines are likely to be uncommented this year. If not I am not convinced that they are complicated lines that need to be saved for when they become useful. If we decide to handle the user's continuation lines then the method of doing that may change. If we want to keep these lines in comments then they also need to be reviewed (and I have comments)

Reply:
yes, they are not that important we can delete them

Reply:
done